apps/server/src/connect/connectApi.ts

```diff
@@ -29,6 +32,24 @@ function mapInstitution (ins: Institution) {
   })
 }
 
+function mapInstitutionLocal (ins: LocalInstitution): InstitutionResponse {
```

Comment:
Why don't we modify mapInstitution instead of making mapInstitutionLocal? I think we want the code to behave exactly the same regardless of where the cached data is populated from

Reply:
Both methods are needed because one is for mapping institutions which come from elasticsearch results and one maps institutions which come from provider apis and the data formats are different between them.

Reply:
The improved naming helps